### ISSUE TITLE
Clear performance marker on cancel/error

### ISSUE THREAD

**Issue by johnwcallahan** [Zizzamia/perfume.js#71]
Thanks for the great library! 

I'm looking for a way to clear (i.e. cancel) a performance marker. My use case is that I don't want to log a timing event if what I'm measuring is interrupted (or fails). Here's a simple example using React:

```jsx
class MyComponent extends React.Component {
  constructor() {
    perfume.start('MeasureMyComponent');
  }

  componentDidMount() {
    loadData().then(() => {
      perfume.end('MeasureMyComponent');
    }).catch(err => {
      // ... clear the marker if there's an error
    });
  }

  componentWillUnmount() {
    // ... clear the marker if the component unmounts before loadData() is done
  }
}
```

Is there a way to do this with perfume.js? I haven't been able to find it in the docs or source code, but maybe I'm missing something.

**Comment by Zizzamia** [comment]
Thank you for the issue.
That's something I didn't think about it before, but it seems I should create a `perfume.clear('MeasureMyComponent')` based on the official Performance API.
https://developer.mozilla.org/en-US/docs/Web/API/Performance/clearMarks

**Comment by Zizzamia** [comment]
This feature will be available with [v4.4.0](https://github.com/Zizzamia/perfume.js/releases/tag/v4.4.0).
Really appreciated the feedback 🔥🔥🔥
